tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -90,7 +89,7 @@ def resolve_launch_arguments(args):
     # try to resolve launch file in package first
     if len(args) >= 2:
         try:
-            resolved = roslaunch.resource_cache.find_resource(args[0], args[1])
+            resolved = roslib.packages.find_resource(args[0], args[1])
```

Comment:
Can we use cached resource for find_resource? (not only find_node at https://github.com/ros/ros_comm/pull/682/files#diff-67998749e41c63572aa0901b6715e9a0R251)